src/UnitTestExecutor.cpp

```diff
@@ -44,8 +47,11 @@ void UnitTestExecutor::executeAllTests() {
 */
 GenericUnitTester* UnitTestExecutor::getUnitTester(const GUID& guid) {
 	CDynamic_Library *library = new CDynamic_Library;
+	logger.info(L"Creating dynamic library.");
 	TestFilter *testFilter = new TestFilter;
+	logger.info(L"Creating test filter.");
 
 	GenericUnitTester* unitTester = GuidTesterMapper::GetInstance().getTesterInstance(library, testFilter, &guid);
+	logger.info(L"Getting unit tester.");
```

Comment:
Tohle už je v podstatě dvakrát v `GuidTesterMapperu`, takže tady to je taky zbytečný.